packages/common/src/common.ts

```diff
@@ -779,7 +723,8 @@ export class Common {
       const blockOrTime = hf.timestamp ?? hf.block
       if (
         (hf.forkHash === null || hf.forkHash === undefined) &&
-        ((blockOrTime !== null && blockOrTime !== undefined) || typeof hf.ttd !== 'undefined')
```

Comment:
Q: how are we preventing paris from not adding to the forkhash, may be we should check for paris here?

Reply:
This is actually handled in the `_calcForkHash()` method, and we do have a `paris` check already in there, just looked it up (and tested via REPL):

```ts
if (
  typeof blockOrTime === 'number' &&
  blockOrTime !== 0 &&
  blockOrTime !== prevBlockOrTime &&
  name !== Hardfork.Paris
) {
  const hfBlockBytes = hexToBytes(`0x${blockOrTime.toString(16).padStart(16, '0')}`)
  hfBytes = concatBytes(hfBytes, hfBlockBytes)
  prevBlockOrTime = blockOrTime
}
```

So this should not be an issue.